### ISSUE TITLE
[DOC-FIX]

### ISSUE THREAD

**Issue by DhavalRepo18** [mlflow/mlflow#8393]
### Willingness to contribute

Yes. I can contribute a documentation fix independently.

### URL(s) with the issue

This is a documentation for latest MLFlow for deploying model as a container using MLServer. 

https://mlflow.org/docs/latest/models.html#serving-with-mlserver

The guide suggests to use 

```
mlflow models build -m my_model --enable-mlserver -n my-model
```

However, "build" option is not available as a result the output is

```
Error: No such command 'build'.
```

Suggested Fix. build should be replaced by **build-docker**

```
mlflow models build-docker -m my_model --enable-mlserver -n my-model
```

### Description of proposal (what needs changing)

Only Documentation need to be changed as suggested above.

**Comment by dbczumar** [comment]
Hi @DhavalRepo18 , thank you for identifying this issue in our documentation. We would be thrilled about a contribution for this fix. Thank you in advance for your pull request, and please let me know if you have any questions :)

**Comment by vijethmoudgalya** [comment]
Hey as i still see that the docs are still not fixed , can you assign it to me?

**Comment by dbczumar** [comment]
Hi @vijethmoudgalya , absolutely. Feel free to file a PR :)

**Comment by vijethmoudgalya** [comment]
Hey created PR #8480

**Comment by vijethmoudgalya** [comment]
fix #8480

**Comment by harupy** [comment]
Fixed by @vijethmoudgalya

**Comment by harupy** [comment]
Fixed by #8480
